### DISCUSSION TITLE
How do I extract "AssetBundleManifest" files

### DISCUSSION THREAD

**Discussion by Clockwork-LeyLine** [AssetRipper/AssetRipper#902] [Q&A]
I opened an .ab file. There is a file of type AssetBundleManifest(Asset Type ID: 290) in this .ab file. This AssetBundleManifest file contains a list of resources and hash values for the entire game. On the right side of the software, in the Yaml view, there are over 5000 pieces of data for AssetBundleNames and AssetBundleInfos, each of which can be clicked to view. I want to export this data. I use Export all Files, but there is no such data in the exported files. How can I export this data?

**Comment by ds5678** [comment]
Currently, you can't. AssetRipper is programmed to ignore those assets during export because they have no relevance in a Unity project.

**Reply by Clockwork-LeyLine** [reply]
Thank you for your reply. This is really bad news. Some games require information from AssetBundleManifest to get the .ab files. It's really uncomfortable to click on thousands of pieces of information and record them one by one. Hope to add a switch to decide whether to ignore these assets.

**Reply by ds5678** [reply]
If you remove this line, they will export:

https://github.com/AssetRipper/AssetRipper/blob/3c4dbf1ad9e57f6cf331213ef7a60022684ee465/Source/AssetRipper.Export.UnityProjects/Project/ProjectExporter.cs#L78

Alternatively, you can wait for #504 and #861 to be implemented.
